src/pages/Search/components/Heatmap.styles.js

```diff
@@ -0,0 +1,77 @@
+import styled, { css } from 'styled-components';
+
+export const HeatmapContainer = styled.section`
+  width: 100%;
+  padding: 60px 163px 0 163px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const Grid = styled.div`
+  display: grid;
```

Comment:
Note: If someone asks you in an interview if this app would work in IE11, `display: grid` [might make problems](https://caniuse.com/#search=display%20grid)